src/reset-password.html

```diff
@@ -3,12 +3,15 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signup.css">
     <title>Reset Password</title>
 </head>
 <body>
-    <form>
-        <label>Reset Password</label>
-        <input type="password">
+    <form id="sign-up-form">
```

Comment:
Any reason why adding an id here? Curious why you named sign up and not reset?

Usually we'd add an id if it's needed by something, like css or javascript.

Did you mean to reuse the same style from sign up form? In that case I suggest creating a class that uses same style. Or you could define a different id in css with the same css property. I suppose you're just trying to control the width of the form? Maybe a class is desired.

Reply:
Yes, I was trying to use the signup css page for the password page. Just to confirm, I should:

1) change id to class 
2) create that class in a new CSS file for the password page

Let me know if I am missing anything else!

Also adjusted the original request to link to the issue number (thank you for that I forgot...)

Reply:
Yeah, I guess for now we can have a separate class for this form that does same max width.

Going forward if we wanted to generalize to a container class we'd have to decide if all pages would conform to the limited width or not. Whether they'd be responsive in the sense the width would change depending on screen size (mobile, tablet, laptop, etc).

Reply:
Okay! I wanted to add as well that I added CSS to make the button match what we have in other files.